packages/editor-ui/src/components/Node/NodeCreation.vue

```diff
@@ -98,59 +85,61 @@ function nodeTypeSelected(nodeTypes: string[]) {
 	emit('addNodes', getAddedNodesAndConnections(nodeTypes.map((type) => ({ type }))));
 	closeNodeCreator(true);
 }
+
+onMounted(() => {
+	wrapperBoundingRect.value = wrapperRef.value?.getBoundingClientRect();
+
+	document.addEventListener('mousemove', onMouseMove);
+});
+
+onBeforeUnmount(() => {
+	document.removeEventListener('mousemove', onMouseMove);
+});
 </script>
 
 <template>
-	<div>
```

Comment:
Diff is weird. Only deleted wrapper `<div>`, added `wrapperRef` and removed the `noEvents` class.